.github/workflows/release-selfhost.yml

```diff
@@ -108,8 +108,8 @@ jobs:
       - name: Perform Github Release
         uses: softprops/action-gh-release@v1
         with:
-          name: v${{ env.RELEASE_VERSION }}
-          tag_name: v${{ env.RELEASE_VERSION }}
+          name: ${{ env.RELEASE_VERSION }}
+          tag_name: ${{ env.RELEASE_VERSION }}
```

Comment:
Thank's for having a look at this. I was just about to take it on but you've saved me the trouble 👏 
Should we update the default tag behaviour to remove the `v` also then?
https://github.com/Budibase/budibase/blob/a0202a7e48b38b690dd2b013cb3dac367cbe5c3d/scripts/versionCommit.sh#L16

Reply:
I think the github tag actually determines the image tag, so we will have to update this